content/blog/new-powershell-library-for-the-hpe-greenlake-cloud-platform.md

```diff
@@ -0,0 +1,405 @@
+> Get-Command -Module HPEGreenLake
+```
+
+In this first release, about 50 cmdlets are available with the HPE GreenLake module. The table below describes the main resources and features that are supported:
+
+![](/img/LJ-key-supported-resources.png)
+
+In PowerShell, cmdlet names are constructed using a verb-noun format. The verb describes the action that the cmdlet performs (Get, Set, Remove, Invoke, etc.), and the noun specifies the object that the cmdlet acts upon. For example, the **Get-HPEGLUserRole** retrieves information about the user role in the HPE GreenLake platform. Note that object resource with the HPE GreenLake library always starts with **HPEGL<resource_name>** (GL for GreenLake). 
+
+**Get-Help** is an important cmdlet in PowerShell as it provides detailed information about a specific cmdlet or function. To get the full help details for the **Get-HPEGLUserRole** cmdlet, you can use the following command:
+
+```powershell
+> Get-Help Get-HPEGLUserRole -Full
+```
+
+To view the detailed examples of how to use a particular cmdlet, you can use the **Get-Help** cmdlet along with the **\-Examples** parameter followed by the name of the cmdlet. Here's an example command that you can use to get the examples of **Get-HPEGLUserRole** cmdlet:
+
+```powershell
+> Get-Help Get-HPEGLUserRole -Examples
+```
+
+This will display all the available examples for the **Get-HPEGLUserRole** cmdlet in a list format. You can review the examples and use them according to your requirements.
+
+# Connection to the HPE GreenLake platform
+
+The connection to the HPE GreenLake platform is done using the **Connect-HPEGL** cmdlet.
+
+>**Note**: The library currently only supports single-factor authentication. Multi-factor authentication (MFA) and SAML Single Sign-On are not supported. 
+
+>**Note**: This limitation means that users who use SAML single sign-on with the HPE GreenLake platform (this applies to all HPE employees) cannot use their corporate email credentials when logging in via the **Connect-HPEGL** cmdlet. 
+
+> **Note**: While waiting for SAML Single Sign-On support, the temporary solution is to add a secondary email into your HPE GreenLake account. Just go to the HPE GreenLake GUI and use the **Invite Users** card in **Manage** / **Identity & Access** to send an invitation to a non-corporate email address. Once you receive the email, accept the invitation and you will be directed to the HPE GreenLake interface where you can set a password. Once this is done, you can use this email address and password to log in with **Connect-HPEGL**.
+
+> **Note**: To interact with the HPE GreenLake platform through this library, you must possess at least the ***Observer*** role in the ***HPE GreenLake platform*** application. This role only grants view privileges. However, if you need to make modifications, then either the ***Operator*** (with view and edit privileges) or the ***Administrator*** (with view, edit, and delete privileges) roles are necessary.
```

Comment:
Each user invited to a workspace is attached to a role.
I would prefer to talk about these as built-in roles, and add an sentence to highlight custom roles for GreenLake Platform can be created. Such as "or create your own custom role which fit your role based access control requirement"

Reply:
Just checked, and you are right! I will update my comment!